dual_net.py

```diff
@@ -123,30 +123,21 @@ def get_inference_input():
              'value_tensor': tf.placeholder(tf.float32, [None])})
 
 
-def model_fn(features, labels, mode):
-    '''
+def model_inference_fn(features, training):
+    """Builds just the inference part of the model graph.
+
     Args:
-        features: tensor with shape
-            [BATCH_SIZE, go.N, go.N, features_lib.NEW_FEATURES_PLANES]
-        labels: dict from string to tensor with shape
-            'pi_tensor': [BATCH_SIZE, go.N * go.N + 1]
-            'value_tensor': [BATCH_SIZE]
-        mode: a tf.estimator.ModeKeys (batchnorm params update for TRAIN only)
-    Returns: tf.estimator.EstimatorSpec with props
-        mode: same as mode arg
-        predictions: dict of tensors
-            'policy': [BATCH_SIZE, go.N * go.N + 1]
-            'value': [BATCH_SIZE]
-        loss: a single value tensor
-        train_op: train op
-        eval_metric_ops
-    return dict of tensors
-        logits: [BATCH_SIZE, go.N * go.N + 1]
-    '''
+        features: input features tensor.
```

Comment:
keep the shape comments OR move the def of `model_fn` above `model_inference_fn` in the file?

Reply:
moved `model_inference_fn` down